dlt/extract/decorators.py

```diff
@@ -642,7 +648,9 @@ def make_resource(_name: str, _section: str, _data: Any) -> TDltResourceImpl:
             selected,
             cast(DltResource, data_from),
             True,
+            incremental=incremental,
```

Comment:
IMO we should extend `make_hints` to accept incremental config. then we'll be able to use existing mechanism that injects incremental into functions